scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -119,6 +110,32 @@ object AcceptAllPathFilter extends PathFilter {
 }
 
 object FileSource {
+  val LOG = LoggerFactory.getLogger(this.getClass)
+
+  val VerboseFileSourceLoggingKey = "com.twitter.scalding.filesource.verbose.logging"
+
+  def verboseLogEnabled(conf: Configuration): Boolean = conf.getBoolean(VerboseFileSourceLoggingKey, false)
+
+  def ifVerboseLog(conf: Configuration)(msgFn: => String): Unit = {
```

Comment:
these can actually all be private, or maybe private[scalding] -- would that be enough?